BUILD

```
@@ -1,3 +1,8 @@
+load(
+    "//source/extensions:extensions_build_config.bzl",
```

```suggestion
    "@envoy_build_config//:extensions_build_config.bzl",
```
Is the real overridden path :)